BootloaderCorePkg/Tools/ConfigEditor.py

```diff
@@ -624,6 +624,8 @@ def update_widgets_visibility_on_page(self):
     def combo_select_changed(self, event):
         self.update_config_data_from_widget(event.widget, None)
         self.update_widgets_visibility_on_page()
+        self.frame_right.update()
+        self.conf_canvas.config(scrollregion=self.conf_canvas.bbox("all"))
```

Comment:
Maybe better to call update_page_scroll_bar() instead ?

If you want to keep the cursor at current selected item, we can add a parameter to update_page_scroll_bar() to control if we need to reset the yview position to 0 or not.

Reply:
Sure, added parameter in update_page_scroll_bar.